client/src/components/App.jsx

```diff
@@ -63,27 +63,28 @@ class App extends React.Component {
   }
 
   render() {
+    let { currentProductID, currentProduct, productStylesArray, currentStyleID, changeCurrentStyle, questionList, reviews, relatedItems  } = this.state;
     if (this.state.currentProductID) {
       return (
-        <div>
+        <div className="container">
```

Comment:
What happened to the container separating your components from the rest?

Reply:
This container wrapped everything but the Nav bar and my component, which was causing a centering issue. I did not see a reason to keep them in separate. Is there one that you know of?